src/orchestration/constants.ts

```diff
@@ -0,0 +1,12 @@
+export enum PAGE_ID_FORMAT {
```

Comment:
qhanam@, do you think `src/orchestration/constants.ts` appropriate naming/place to hold the enums used in Orchestration and test cases?

Reply:
I think these should stay in Orchestration, as they are part of the configuration.

We can also export them from `index.ts` so they are available to consumers of the web client.